lib/src/page_route.dart

```diff
@@ -32,6 +33,14 @@ class SwipeablePageRoute<T> extends CupertinoPageRoute<T> {
   ///
   /// Set this to `false` to disable swiping completely.
   bool canSwipe;
+  
+  /// Forward transition duration. Default is CupertionPageRoute transitionDuration
+  Duration? _transitionDuration;
+  
+  @override
+  Duration get transitionDuration {
+    return _transitionDuration ?? super.transitionDuration;
+  }
```

Comment:
This removes the indentation of blank lines and makes `_transitionDuration` final as it's private and not changed after instantiation.

```suggestion
  /// An optional override for the [transitionDuration].
  final Duration? _transitionDuration;

  @override
  Duration get transitionDuration =>
      _transitionDuration ?? super.transitionDuration;
```